keyboards/darkproject/kd87b_ltd/info.json

```
@@ -0,0 +1,280 @@
+{
+    "keyboard_name": "KD87b LTD",
+    "manufacturer": "Dark Project",
+    "url": "https://darkproject.ru",
+    "maintainer": "Gsky",
+    "usb": {
+        "vid": "0x342D",
+        "pid": "0xE393",
+        "device_version": "0.0.3",
+        "force_nkro": true
+    },
+    "indicators": {
+        "caps_lock": "C14"
+    },
+    "processor": "WB32FQ95",
+    "bootloader": "wb32-dfu",
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "build": {
+        "debounce_type": "sym_defer_pk",
+        "lto": true
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["A0", "A1", "A2", "A3", "A4", "A8", "A9", "A10"],
+        "rows": ["B0", "B1", "B2", "B3", "B4", "B5", "B6", "B7", "B8", "B9", "B10", "B12"]
+    },
+    "bootmagic": {
+        "matrix": [1, 3] // Hold ESC on start up to clear EEPROM and boot into bootloader mode
```

please move this comment to `readme.md`
```suggestion
        "matrix": [1, 3]
```